src/Status.cxx

```diff
@@ -0,0 +1,152 @@
+#include <cmath>
+
+#include <rapidjson/document.h>
+
+#include "Status.hpp"
+
+double average(const double &sum, const double &N) { return sum / N; }
+
+double standard_deviation(const double &sum, const double &sum_squared,
+                          const double &N) {
+  auto variance = (sum_squared - (sum * sum) / N) / (N - 1);
```

Comment:
I stumbled again over this formula. I asked in my previous review because I wasn't sure and didn't check myself. But now, when I write it down, I think what you had previously was correct, if the function parameters mean what I think they mean:
`sum_squared = \sum_i x_i^2` and `sum = \sum_i x_i`

Reply:
As we discussed yesterday yes, ``sum_squared = \sum_i (x_i^2)`` and ``sum = \sum_i x_i``